bin/docker-compose.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+set -x
+
+# Pass commands to Docker Compose v1 or v2 depending on what is present
+
+if (docker compose ls >/dev/null); then
```

Comment:
should we just be using docker compose now that docker-compose is deprecated

last update 2021, https://pypi.org/project/docker-compose/#history

Reply:
We could. OTOH `serve.sh` still references `paster` :D